api/tests/unit/infrastructure/serializers/jsonapi/organization-serializer_test.js

```diff
@@ -1,156 +1,100 @@
-const { expect } = require('../../../../test-helper');
+const { expect, factory } = require('../../../../test-helper');
 const serializer = require('../../../../../lib/infrastructure/serializers/jsonapi/organization-serializer');
-const Organization = require('../../../../../lib/domain/models/Organization');
-
-const faker = require('faker');
 
 describe('Unit | Serializer | organization-serializer', () => {
 
   describe('#serialize', () => {
 
-    context('when user is defined', () => {
-
-      it('should serialize organization with included user', () => {
-        // given
-        const jsonOrganization = {
-          id: 12,
-          name: 'LexCorp',
-          type: 'PRO',
-          code: 'ABCD66',
-          userId: '42157',
-          user: {
-            id: 42157,
-            firstName: 'Alexander',
-            lastName: 'Luthor',
-            email: 'lex@lexcorp.com'
-          }
-        };
-        const organization = new Organization(jsonOrganization);
+    it('should return a JSON API serialized organization', () => {
+      // given
+      const organization = factory.buildOrganization();
 
-        // when
-        const serializedOrganization = serializer.serialize(organization);
+      // when
+      const serializedOrganization = serializer.serialize(organization);
 
-        // then
-        expect(serializedOrganization).to.deep.equal({
-          data: {
-            type: 'organizations',
-            id: 12,
-            attributes: {
-              name: 'LexCorp',
-              type: 'PRO',
-              code: 'ABCD66'
-            },
-            relationships: {
-              user: {
-                data: { type: 'users', id: '42157' }
-              },
-            }
+      // then
+      expect(serializedOrganization).to.deep.equal({
+        data: {
+          type: 'organizations',
+          id: organization.id,
+          attributes: {
+            'name': organization.name,
+            'type': organization.type,
+            'code': organization.code,
           },
-          included: [{
-            id: '42157',
-            type: 'users',
-            attributes: {
-              'first-name': 'Alexander',
-              'last-name': 'Luthor',
-              email: 'lex@lexcorp.com'
-            }
-          }]
-        });
+          relationships: {
+            user: { data: null },
+            members: { data: [] }
+          }
+        }
       });
     });
 
-    it('should serialize an array of organization', () => {
+    it('should include serialized user data with organization has a user', () => {
```

Comment:
Il manque un mot dans la phrase : "should include serialized user data with organization IF has a user" ?